Changed the email template as requested.

afzaledx opened the pull request:
Changed the email template as requested.

Commit:
Update instructions in activation e-mail.

pomegranited commented:
@afzaledx Since you're adding a file to the cloudera theme, could you please see [the README](https://github.com/open-craft/edx-theme/tree/cloudera#custom-templates) for further instructions?  We have a short process that helps with future rebases.  Thanks!

Commit:
Add upstream version of lms/templates/emails/activation_email.txt.

Commit:
Merge branch 'cloudera-upstream-templates' into cloudera

itsjeyd commented on lms/templates/emails/activation_email.txt:
```diff
@@ -0,0 +1,21 @@
+
+${_("The {platform_name} Team").format(platform_name=platform_name)}
+
+${_("If you need help, email {support_email}.").format(support_email=support_email)}
```
@afzaledx This is missing a "please" :)

> If you need help, **please** email {support_email}.

Also, does `support_email` include the `mailto` link markup that Cloudera requested? If not, we'll need to add it here as well.

itsjeyd reviewed (changes requested):
@afzaledx Thanks for the changes! I left some feedback that should be quick to address.

afzaledx commented:
@itsjeyd, I have used the support_email, which is coming from CONTACT_EMAIL, so I think this PR is good to go. Anything else I need to fix here?

itsjeyd reviewed (approved):
@afzaledx This LGTM :+1: Before merging please squash commits so that the changes to the upstream version of the template are consolidated in a single commit. To be specific, the resulting list of commits should look like this:

* b9b61edbd835c884a68bb5ab44f96a22cecc6683: Add upstream version of lms/templates/emails/activation_email.txt.
* 937a516ff187f3b5082c3795d848e748144d87c7: Merge branch 'cloudera-upstream-templates' into cloudera
* `<hash of new commit containing changes>`: Update instructions in activation e-mail.

---

Please also push the updates that you made to the [`cloudera-upstream-templates`](https://github.com/open-craft/edx-theme/tree/cloudera-upstream-templates) to GitHub. I noticed that the [documentation](https://github.com/open-craft/edx-theme/tree/cloudera#custom-templates) that @pomegranited linked to earlier doesn't mention this step explicitly, but it's necessary to make the updates available to the entire team.

---

- [x] I ~~tested this~~ will test this once a new app server  with the updated e-mail template is up and running.
- [x] I read through the code
- [x] ~~I checked for accessibility issues~~ N/A
- [x] ~~Includes documentation~~ N/A

afzaledx commented:
@itsjeyd , I didnt really think and just pushed the original template directly to the cloudera-upstream-templates. Would a PR against the {{cloudera-upstream-templates}} have been better (for next time)? Also squashed the commits in this branch, and changed the commit meessage according to your suggestion. Thanks for reviewing this. I am now merging the PR and will create a new server, test it out and let you know.

itsjeyd commented:
Thanks @afzaledx!

> I didnt really think and just pushed the original template directly to the cloudera-upstream-templates. Would a PR against the {{cloudera-upstream-templates}} have been better (for next time)?

Good question :) Looking at the [commits](https://github.com/open-craft/edx-theme/commits/cloudera-upstream-templates) in the `cloudera-upstream-templates` branch it doesn't seem like we're following a PR workflow when adding upstream templates, so it should be fine to keep pushing these types of changes directly in the future. My only suggestion would be to hold off on pushing until it's clear which templates are going to be needed. (There might be cases where, depending on the type and size of a theme change, it's not obvious which templates to change right from the start.)
